microsoft/PowerToys
# Https scheme fix, merging #12790

## Summary of the Pull Request
Merging for #12790 

**What is this about:**
In Powertoy Run some schemes were incorrectly being overwritten with HTTPS. This change fixes this.

**What is include in the PR:** 
Should only be changes to 2 files

**How does someone test / validate:** 
Execute Powertoy Run, type `mailto:test@example.com`, you should see the correct scheme (mailto) in the drop down.

## Quality Checklist

- [x] **Linked issue:** #12764

## Contributor License Agreement (CLA)
A CLA must be signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/PowerToys) and sign the CLA.
